### DISCUSSION TITLE
蓝叠国际版无法连接问题

### DISCUSSION THREAD

**Discussion by sayasora** [MaaAssistantArknights/MaaAssistantArknights#11173] [General]
我使用的是蓝叠国际版，但maa一直连接失败，反复修改adb相关设置后，发现是模拟器没有开启相关功能，可能其他模拟器是自动开启的，蓝叠国际版需要在设置中手动打开
![image](https://github.com/user-attachments/assets/5c16f00f-a7cd-48f2-8742-19fff0a43b76)
![image](https://github.com/user-attachments/assets/f5612337-a65f-4806-99b4-aa2791c7cd1b)

**Comment by ABA2396** [comment]
![image](https://github.com/user-attachments/assets/4b809d1e-fc1d-468a-96c2-39bdf8e3f9ec)

**Reply by sayasora** [reply]
不好意思，因为当时maa报错是检查连接及adb相关设置，所以查询的手册中的《连接设置》，没想到在《模拟器和设备支持中》
